packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -395,7 +392,7 @@ export function requestUpdateLane(fiber: Fiber): Lane {
   if ((mode & BlockingMode) === NoMode) {
     return (SyncLane: Lane);
   } else if ((mode & ConcurrentMode) === NoMode) {
-    return getCurrentPriorityLevel() === ImmediateSchedulerPriority
```

Comment:
Notable update here - now we can just check if the `updateLanePriority` is `Sync`, otherwise use batched.